### PR TITLE
bugfix: make `parallel_tool_calls` be a public member

### PR DESCRIPTION
**Pull request by cdxker**
Two bugfixes:
- One is from the issue seen in `ChatCompletionParameters` and using `..Default::default()`. I'm not sure why `parallel_tool_calls` is a private member as it doesn't seem to be used anywhere internally. 

image

- The other is from gemini-flash returning `STOP` instead of `stop` as the stop token

### CODE REVIEW

**Comment by tjardoo** [comment]
Thanks! Maybe you can also add the other Gemini finish reasons, so they're all covered.

**Comment by cdxker** [comment]
Hey sorry, just saw this. Added more `FinishReasons`.

File: openai_dive/src/v1/resources/shared.rs [hunk 1]
```diff
@@ -136,18 +136,32 @@ pub enum LastErrorCode {
...
-    #[serde(rename = "stop")]
+    #[serde(rename = "stop", alias = "STOP")]
     StopSequenceReached,
     /// Incomplete model output due to max_tokens parameter or token limit.
-    #[serde(rename = "length")]
+    #[serde(rename = "length", alias = "MAX_TOKENS")]
     TokenLimitReached,
     /// Omitted content due to a flag from our content filters.
-    #[serde(rename = "content_filter")]
+    #[serde(
```
Comment (cdxker) [inline openai_dive/src/v1/resources/shared.rs:145]: image

Bundled all of these together just to limit having multiple enums to handle. According to [Gemini docs](https://cloud.google.com/vertex-ai/generative-ai/docs/reference/python/latest/vertexai.preview.generative_models.FinishReason?authuser=1) seems like they all go under the same category.

File: openai_dive/src/v1/resources/shared.rs [hunk 2]
```diff
@@ -136,18 +136,32 @@ pub enum LastErrorCode {
...
+        rename = "content_filter",
+        alias = "SAFETY",
+        alias = "SPII",
+        alias = "PROHIBITED_CONTENT",
+        alias = "BLOCKLIST",
+        alias = "RECITATION"
+    )]
     ContentFilterFlagged,
```
Comment (cdxker) [inline openai_dive/src/v1/resources/shared.rs:153]: image

Bundled all of these together just to limit having multiple enums to handle. According to [Gemini docs](https://cloud.google.com/vertex-ai/generative-ai/docs/reference/python/latest/vertexai.preview.generative_models.FinishReason?authuser=1) seems like they all go under the same category.

**Pull request merged**